doc-site/docs/getting-started/installation-advanced.md

```diff
@@ -192,16 +217,53 @@ smartContractsReferences:
     address: "0x4456307ef3f119dac17a5e974d2640f714e6edb0"
 
 paladinNodes:
-  - name: branch-a
-    baseLedgerEndpoint:
-      type: endpoint
-      endpoint:
-        jsonrpc: http://besu1:8545
-        ws:    ws://besu1:8546
-    paladinRegistration:
-      registryAdminNode: admin
-      registryAdminKey:  registry.operator
-      registry:         evm-registry
+paladinNodes:
+- name: user
```

Comment:
not the best name for a Paladin node, plus it sounds like it's a special required value. something like "Bank-A"?